sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -84,10 +77,6 @@ public final class AzureMonitorExporterBuilder {
     private ConnectionString connectionString;
     private TokenCredential credential;
 
-    // suppress warnings is needed in ApplicationInsights-Java repo, can be removed when upstreaming
-    @SuppressWarnings({ "UnusedVariable", "FieldCanBeLocal" })
-    private AzureMonitorExporterServiceVersion serviceVersion;
```

Comment:
This field was not used.

Reply:
@lmolkova I remember the Azure SDK folks asking specifically for this, do you know if it's ok that we're not using it / removing it?

Reply:
If I remember correctly the concerns were for quick pulse REST API definitions. Here given that service does not have a notion of api version, it's best to remove it.